addons/social_share/models/snshare_template_group.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+class CampaignTemplate(models.Model):
+    _name = 'snshare.template.group'
```

Comment:
As per discussion, this seems a bit overkill for a first implementation, heard it was going to disappear so voila.